gui/default/assets/lang/lang-en.json

```diff
@@ -313,15 +327,21 @@
     "Received data is already encrypted": "Received data is already encrypted",
     "Recent Changes": "Recent Changes",
     "Reduced by ignore patterns": "Reduced by ignore patterns",
+    "Registration aborted by user.": "Registration aborted by user.",
+    "Registration failed - possibly aborted, timed out, or blocked by the browser.": "Registration failed - possibly aborted, timed out, or blocked by the browser.",
     "Relay LAN": "Relay LAN",
     "Relay WAN": "Relay WAN",
     "Release Notes": "Release Notes",
     "Release candidates contain the latest features and fixes. They are similar to the traditional bi-weekly Syncthing releases.": "Release candidates contain the latest features and fixes. They are similar to the traditional bi-weekly Syncthing releases.",
+    "Reload page at {%webauthnAddress%}": "Reload page at {{webauthnAddress}}",
+    "Reload page at {%webauthnOrigin%}": "Reload page at {{webauthnOrigin}}",
+    "Reload page at {%webauthnOrigin%} to enable WebAuthn.": "Reload page at {{webauthnOrigin}} to enable WebAuthn.",
```

Comment:
These should be reconciled.  Seems like one or two should suffice.

Reply:
I deduplicated the two that were identical save the interpolation variable name. I don't think eliminating the "Save settings and reload..." is practical - that would mean having to also remove the button and instead instruct the user to first save and then come back to use the remaining "Reload ..." button. But that would still be another translation string, then, just as plain text rather than a button label. Is deduplicating the obvious one enough?

Reply:
(That was fixed in 76206b3421c3df8c7b085632977e8b7ab7b51bcd)